app/src/main/java/com/nervousfish/nervousfish/activities/WaitActivity.java

```diff
@@ -100,7 +102,7 @@ protected void onActivityResult(final int requestCode, final int resultCode, fin
     protected void onStart() {
         super.onStart();
         if (this.dataReceived != null && this.tapCombination != null) {
-            evaluateData();
+            this.goToMainActivity();
```

Comment:
Okay, then `evaluateData` maybe was a bad name... Can you edit the log in `goToMainActivity` (since it doesn't seem to be evaluating any data)?